packages/office-addin-debugging/src/port.ts

```diff
@@ -71,7 +75,7 @@ export function getProcessIdsForPort(port: number): Promise<number[]> {
           lines.forEach((line) => {
             /* eslint-disable-next-line @typescript-eslint/no-unused-vars */
             const [protocol, localAddress, foreignAddress, status, processId] = line.split(" ").filter((text) => text);
-            if (processId !== undefined) {
+            if (processId !== undefined && processId !== '0') {
```

Comment:
My guess is that this `0` only works for Windows right?
Could we add also test on other platforms if possible and, if it doesn't work on them, can we add some `if` for checking if we are on Windows.

Reply:
Can we then make this `0` option only be checked for Win32?
We don't want this to accidentally create some bug on other platforms now or in the future.

Reply:
It already is (if statement starts on line 74)